docs/resources/workflow.md

```diff
@@ -72,3 +73,48 @@ Optional:
 - `reference` (String)
 
 
+
+
+
+<a id="nestedatt--steps"></a>
+### Nested Schema for `steps`
+
+Required:
+
+- `name` (String)
+- `param_bindings` (Attributes List) (see [below for nested schema](#nestedatt--steps--param_bindings))
+
+Optional:
+
+- `for_each` (String)
```

Comment:
We should definitely document these better, can we make sure we have a card to make these really nice?

Reply:
we haven't linked to the API docs just yet. I found it very fiddly due to the number of subtly differently named payload types

I've included it in a polish ticket which also covers ironing out the other fields (bool flags etc) https://linear.app/incident-io/issue/CAT-275

Reply:
tbh I would just write it out, I think using the docs is more effort than it's worth here